CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+
+Note: For merge commits, just use `chore` for the type of change, `merge` for the scope, and `merge commit` for the message. Likely, however, it will not matter what you put into Commitizen as you'll just get a normal merge commit message such as `Merge branch 'my-branch' into canary`.
+
+## Git Workflow and Branches
+Specific branches will be used for specific purposes in this repo. Considering this, the lifecycle of a new feature, bugfix, etc. should largely follow the steps below.
+
+### If you are a community contributor:
+1. Fork the Irving repo.
+2. Create a new branch based on `master`. If your branch relates to a specific github issue, reference that issue in your branch name. Example: `feature/issue-17-branch-title` where `branch-title` should correspond rougly to the topic of the issue you're addressing.
+3. If you've fixed a bug or added a new feature, please write tests! Ensure the tests pass using `npm run test` and/or `npm run test:watch` (for running tests on change). These commands can be run from within the package you are modifying or at the root of the repo if you want to run all tests for all packages.
+4. Work on your brach, using `npm link` where necessary to test out your code.
+5. When ready, push up your branch and create a pull request into `master`. Your code will be reviewed by someone at Alley.
+6. Assuming your branch passes review, someone at Alley will merge it.
+7. Your code should be availabile on npm in the next prerelease.
+8. Keep an eye on the releases page. When you see your code has been release, _install and test it!_.
+9. Assuming neither you nor someone at Alley finds issues with your code, it will also be included in the next production release.
+
+### If you are a part of the Alley organization and have publish capabilities for the npm packages:
+1. Clone the irving repo
+2. Create a new branch based on `master`. If your branch relates to a specific github issue, reference that issue in your branch name (and all subsequent commits).
+* Example: `feature/issue-17-branch-title` where `branch-title` should correspond rougly to the topic of the issue you're addressing.
+* Defer to using the issue number over a Jira ticket number to keep pull requests and commits consistent.
+3. Work on your branch, using `npm link` where necessary.
+4. Merge at will into the `canary` branch and test using `npm link` and/or `npm run release:canary` to release on the `@canary` tag.
+5. After testing to your heart's content, push your branch and create a pull request into master for a code review.
+6. Once your code passes review, merge it into `master` when ready.
+6. Your code should be availabile on npm in the next prerelease.
+7. Keep an eye on the releases page. When you see your code has been release, _install and test it!_.
```

Comment:
These steps don't apply to Alley devs who have presumably created their own prerelease, right?